src/components/Register.module.css

```diff
@@ -0,0 +1,39 @@
+.headerBackground {
+  background-image: url(../assets/LAPalmTrees.png);
+  height: 250px;
+  background-size:cover;
+  min-width: 100%;
+}
+
+.card {
+  position: relative;
+}
```

Comment:
This style isn't used so can be removed.

Reply:
done